lib/main.dart

```diff
@@ -26,18 +26,80 @@ void main() {
       [DeviceOrientation.portraitUp, DeviceOrientation.portraitDown]);
 
   // run app
-  runApp(new GameContainer());
```

Comment:
adding `RawKeyboardListener` as a wrapper around GameContainer(), does seem like it can work, but there is a lot of useless code added to this file and it might not be the cleanest, since event would need to propagte into GameContainer, so why not so it in game directly.